Frameworks/UIKit/UIPopoverPresentationController.mm

```diff
@@ -316,7 +325,7 @@ - (void)setPopoverBackgroundViewClass:(Class)popoverBackgroundViewClass {
 }
 
 - (void)dealloc {
-    [_willDismissCompletion release];
+    [_onDismissCleanupHandler release];
```

Comment:
Handled by strong id change. The dismissCompletion won't be invoked prior to dealloc though (this is ensured by grabbing the strong ref in _handleDismiss)